logstash-core/lib/logstash/pipeline.rb

```diff
@@ -38,6 +38,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
yes the argument for not using false there is the need for that message. I guess this can be refactored in another way, potentially using exceptions, but I guess for now it ok.